mods/ra/maps/allies-02/allies02.lua

```diff
@@ -241,11 +241,19 @@ WorldLoaded = function()
 
 	Trigger.AfterDelay(DateTime.Seconds(1), function() Media.PlaySpeechNotification(allies, "MissionTimerInitialised") end)
 
-	RunInitialActivities()
-
-	Reinforcements.Reinforce(player, ConstructionVehicleReinforcements, ConstructionVehiclePath)
-	Trigger.AfterDelay(DateTime.Seconds(5), SendJeepReinforcements)
-	Trigger.AfterDelay(DateTime.Seconds(10), SendJeepReinforcements)
+	--RunInitialActivities()
+
+	--Reinforcements.Reinforce(player, ConstructionVehicleReinforcements, ConstructionVehiclePath)
```

Comment:
Why comment this out instead of deleting it and retrieving it from history if it is ever needed again?